test/models/gemini_connection_test.rb

```diff
@@ -27,6 +27,16 @@ class GeminiConnectionTest < SidekiqTestCase
           assert_instance_of(klass, conn.refresh_authorization!)
         end
       end
+
+      describe "when unsuccessful" do
+        before do
+          mock_unknown_failure(klass.oauth2_client.token_url)
+        end
+
+        test "it should raise an error" do
+          assert_raises(Oauth2::Errors::UnknownError) { conn.refresh_authorization! }
```

Comment:
Can we test that the value is raised. It looks like we're testing for the thing we deleted.

Reply:
@tsmartt No, the failing error was "ArugmentError"

Reply:
`Loading development environment (Rails 6.1.4.7)
irb(main):001:0> raise Oauth2::Errors::UnknownError
/var/www/lib/oauth2/errors.rb:6:in `initialize': missing keywords: :response, :request (ArgumentError)
irb(main):002:0>`